python/ml4ir/applications/ranking/tests/data/configs/model_config_auto_dag_network.yaml

```diff
@@ -16,7 +16,7 @@ layers:
     args:
       units: 2
       activation: relu
-  - type: keras.layers.merge.Concatenate
+  - type: keras.layers.merging.concatenate.Concatenate
```

Comment:
Nit: I don't see a change linked with concatenate layers in the release notes.

Reply:
This is just a config update in one of the tests. The actual change seems to have happened on the keras side. They've renamed `layers.merge` -> `layers.merging.concatenate`